Classes/Command/IndexCommand.php

```diff
@@ -87,13 +87,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
             exit(1);
         }
 
-        if ($input->getOption('solr')) {
+        if (
+            !empty($input->getOption('solr'))
+            && !is_array($input->getOption('solr'))
```

Comment:
`$input->getOption('solr')` is never an array. This is garantied by symfony. 
To accept arrays, we must add `InputOption::VALUE_IS_ARRAY` which is not what we want